crates/edr_provider/src/lib.rs

```diff
@@ -380,10 +382,12 @@ impl<LoggerErrorT: Debug + Send + Sync + 'static, TimerT: Clone + TimeSinceEpoch
             // debug_* methods
             MethodInvocation::DebugTraceTransaction(transaction_hash, config) => {
                 debug::handle_debug_trace_transaction(data, transaction_hash, config)
+                    .and_then(normalise_rpc_debug_trace)
```

Comment:
Please move this call inside the `handle_debug_trace_transaction` call, following the same pattern as all other functions. The `handle_*` functions determine the result type, not the provider.